src/css/banner.css

```diff
@@ -0,0 +1,3 @@
+.banner {
+  font-weight: bold;
```

Comment:
Why haven't you used tailwind font weight style ?

Reply:
will use it :+1: